cli/xml2rfc/writers/preptool.py

```diff
@@ -1282,18 +1291,18 @@ def references_check_usage(self, e, p):
     #       pn='p-[section]-[counter]'
     def boilerplate_section_add_number(self, e, p):
         self.boilerplate_section_number += 1
-        e.set('pn', '%s-boilerplate.%s' % (pnprefix[e.tag], self.boilerplate_section_number, ))
+        self.set_element_pn(e, '%s-boilerplate.%s' % (pnprefix[e.tag], self.boilerplate_section_number, ))
 
     def toc_section_add_number(self, e, p):
         self.toc_section_number += 1
-        e.set('pn', '%s-toc.%s' % (pnprefix[e.tag], self.toc_section_number, ))
+        self.set_element_pn(e, '%s-toc.%s' % (pnprefix[e.tag], self.toc_section_number, ))
 
     def front_abstract_add_number(self, e, p):
-        e.set('pn', '%s-abstract' % pnprefix[e.tag])
+        self.set_element_pn(e, '%s-abstract' % pnprefix[e.tag])
 
     def front_note_add_number(self, e, p):
         self.note_number += 1
-        e.set('pn', '%s-note.%s' % (pnprefix[e.tag], self.note_number, ))
+        self.set_element_pn(e, '%s-note.%s' % (pnprefix[e.tag], self.note_number, ))
```

Comment:
Is it worth noting that this tag doesn't match the comment above on line 1282?

Reply:
Probably a good idea. Those are from an old document that disagrees in more than a few places.